WireAnalytics/Sources/WireAnalytics/Events/Segmentation/SegmentationEntry.swift

```diff
@@ -24,7 +24,26 @@
 
 public struct SegmentationEntry: Hashable, Sendable {
 
-    let key: String
-    let value: String
+    public var key: String
+    public var value: String
```

Comment:
To make it a `var` was probably not necessary and I will revert it.
It's a habit since many times it makes little sense to me to have this restriction on a value-type.

Reply:
Reverted.